salt/_auth/kubernetes_rbac.py

```diff
@@ -148,23 +210,13 @@ def _load_kubeconfig(opts):
     return kubeconfig
 
 
+@_check_auth_args()
```

Comment:
When calling with an incorrect combination of arguments, shouldn't this `raise` instead of return `False` as it does now?

Reply:
Done